detectors/tfoda_new.py

```diff
@@ -0,0 +1,81 @@
+"""
+Perform detection using models created with the Tensorflow Object Detection API.
+https://github.com/tensorflow/models/tree/master/research/object_detection
+This class uses tensorflow to load and run the models, as opposed to tfoda.py
+which uses OpenCV to load and run the TF models.
+"""
+import numpy as np
+import os
+from pathlib import Path
+import sys
+import tarfile
+import tensorflow as tf
+import zipfile
+import time
+
+def scale_box_coords(box, img_w, img_h):
+    res = []
+
+    # The box is in normalised order [ymin, xmin, ymax, xmax]
+    res.append(box[1] * img_w) #xmin
+    res.append(box[0] * img_h) #ymin
+    res.append((box[3] - box[1]) * img_w) #width
+    res.append((box[2] - box[0]) * img_h) #height
+
+    return res
+
+with open(os.getenv('TFODA_CLASSES_PATH'), 'r') as classes_file:
+    CLASSES = dict(enumerate([line.strip() for line in classes_file.readlines()]))
+
+confidence_threshold = float(os.getenv('TFODA_CONFIDENCE_THRESHOLD'))
+model_dir = os.path.join(os.getenv('TFODA_WEIGHTS_PATH'), "saved_model")
```

Comment:
Shouldn't `saved_model` be in `TFODA_WEIGHTS_PATH`? Remember that anyone with any directory structure should be able to use this without having to change the code.